packages/app-mobile/components/screens/Note/Note.tsx

```
@@ -869,10 +876,10 @@ class NoteScreenComponent extends BaseScreenComponent<ComponentProps, State> imp
 			if (resizeLargeImages === 'alwaysAsk') {
 				const userAnswer = await this.props.dialogs.showMenu(
 					`${_('You are about to attach a large image (%dx%d pixels). Would you like to resize it down to %d pixels before attaching it?', dimensions.width, dimensions.height, maxSize)}\n\n${_('(You may disable this prompt in the options)')}`, [
-						{ text: _('Yes'), id: 'yes' },
-						{ text: _('No'), id: 'no' },
-						{ text: _('Cancel'), id: 'cancel' },
-					]);
+					{ text: _('Yes'), id: 'yes' },
```

ESLint

[error] Expected indentation of 6 tabs but found 5.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 6 tabs but found 5.

(indent)